bazel/aar_with_jni.bzl

```diff
@@ -55,17 +55,19 @@ EOF
 
     native.genrule(
         name = name,
-        srcs = [android_library + "_kt.jar", android_library + ".aar", archive_name + "_jni_unsigned.apk"],
+        srcs = [android_library + "_kt.jar", android_library + ".aar", archive_name + "_jni_unsigned.apk", "//library:proguard_rules"],
```

Comment:
rather than hardcoding this file location into the bazel rule, what do you think about passing this in as an optional parameter?

Reply:
I think we can make this non-optional since we're going to need to have this in our aar

Reply:
I guess my point is that the aar rule is actually general not project or source-specific, but this embeds a project specific location in it.

Reply:
Or rather, it just seems preferable in general to provide source locations as arguments to bazel rules, rather than have them be part of the rule itself.